[2024-12-10T17:29:24.287+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-10T17:29:24.322+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.process_data scheduled__2024-12-10T16:30:00+00:00 [queued]>
[2024-12-10T17:29:24.342+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_inaturalist_data_atropelamento.process_data scheduled__2024-12-10T16:30:00+00:00 [queued]>
[2024-12-10T17:29:24.343+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 4
[2024-12-10T17:29:24.380+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): process_data> on 2024-12-10 16:30:00+00:00
[2024-12-10T17:29:24.412+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=80) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-10T17:29:24.417+0000] {standard_task_runner.py:72} INFO - Started process 82 to run task
[2024-12-10T17:29:24.418+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'get_inaturalist_data_atropelamento', 'process_data', 'scheduled__2024-12-10T16:30:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/inaturalist_dag.py', '--cfg-path', '/tmp/tmpng53v45k']
[2024-12-10T17:29:24.426+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask process_data
[2024-12-10T17:29:24.526+0000] {task_command.py:467} INFO - Running <TaskInstance: get_inaturalist_data_atropelamento.process_data scheduled__2024-12-10T16:30:00+00:00 [running]> on host 7f9b9dece5d8
[2024-12-10T17:29:24.711+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='get_inaturalist_data_atropelamento' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-10T16:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-10T16:30:00+00:00'
[2024-12-10T17:29:24.713+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-10T17:29:24.716+0000] {python.py:240} INFO - Done. Returned value was: {'path': 'inaturalist_data/atropelamento_data/2024/12/10/atropelamento_observations_172924.csv', 'format': 'text/csv'}
[2024-12-10T17:29:24.767+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-10T17:29:24.768+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=get_inaturalist_data_atropelamento, task_id=process_data, run_id=scheduled__2024-12-10T16:30:00+00:00, execution_date=20241210T163000, start_date=20241210T172924, end_date=20241210T172924
[2024-12-10T17:29:24.838+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-10T17:29:24.886+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-10T17:29:24.890+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
